lib/awful/completion.lua

```diff
@@ -154,10 +161,25 @@ function completion.shell(command, cur_pos, ncomp, shell)
         while true do
```

Comment:
> Would be nice to have your example as a test, in case you have not added it already.

That could be difficult, because we can't possibly know what commands are on the users' systems. What if they don't have command "trust" or they have a command "truecrypt"?

However, we could create a small directory tree with some executable files and use a custom `PATH` while executing the consistency / sorting tests. I'll try something and commit it.

> in my Zsh config [...]

I get the impression that ZSH's completion might be a bit more intelligent (and more powerful, as I have heard ...) than bashcomp. Should I switch shells? ;-)